features/facebookImport/src/popUps/infoPopUps/pictures.jsx

```diff
@@ -1,18 +1,18 @@
 import React from "react";
-
 import i18n from "../../i18n.js";
-import BaseInfoPopUp from "../baseInfoPopUp/baseInfoPopUp.jsx";
 
 const PicturesInfoPopUp = () => {
-    const picturesInfoText = [
+    return (
         <>
             <p>{i18n.t("picturesInfoScreen:text1")}</p>
             <p>{i18n.t("picturesInfoScreen:text2")}</p>
-        </>,
-        <p>{i18n.t("picturesInfoScreen:text3")}</p>,
-    ];
-
-    return <BaseInfoPopUp infoChildren={picturesInfoText} />;
+            <div className="separator separator-space"></div>
```

Comment:
This is showing up repeatedly; I can't really say I like it. Can't it be turned into a "splitScreen" or something like that? Or simply a static (and kinda dumb) Separator component?

Reply:
I don't really know. I think Felix would disapprove 😂  I think we don't want to go too deep with components.

On a different note, I was really surprised that we already have a class called `separator` that did exactly what I needed. Don't ask me where it's coming from, because I don't know. Besides components, it's pretty common to have some utility css classes and this looks like a good candidate.

Reply:
Well, there are *two* classes, and they seem to always go together? OTOH, if something is used repeatedly it's, by definition, a component, so why not give it a single source? It wouldn't be the first time @fhd and me would disagree (or the last), but I don't think it would add any complexity (since it's got no business logic whatsoever) and might simplify design and design choices, as well as source: instead of that, you would simply have `<Separator />`

Reply:
I've been taking a look and they don't always go together. However, there was a class "poly-separator". I deleted "separator" and replaced it to avoid duplications.